osu.Framework/Bindables/Bindable.cs

```diff
@@ -245,6 +245,13 @@ public virtual void Parse(object input)
                     Value = t;
                     break;
 
+                case IBindable _:
+                    if (!(input is IBindable<T> bindable))
+                        throw new ArgumentException($"Attempting to parse {input.GetType()}, which has differnet value type from {typeof(T)}", nameof(input));
```

Comment:
has typo, but in general i'd recommend writing this as

```suggestion
                        throw new ArgumentException($"Expected bindable of type {nameof(IBindable)}<{typeof(T)}>, got {input.GetType()}", nameof(input));
```